MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 recursive-include shoop *.jinja *.html *.txt
 recursive-include shoop_tests *.jinja *.html *.txt
+prune shoop/admin/static_src/*/bower_components
+prune shoop/admin/modules/*/bower_components
+prune shoop/*/node_modules
```

Comment:
Now that I think about this more, I'm not sure if those compiled resources should go to sdist at all. Maybe sdist should contain just the files from the repository and let "setup.py install" phase fetch and compile them?